ansible/group_vars/all.yml

```diff
@@ -1214,6 +1214,9 @@ ironic_admin_endpoint: "{{ admin_protocol }}://{{ ironic_internal_fqdn | put_add
 ironic_internal_endpoint: "{{ internal_protocol }}://{{ ironic_internal_fqdn | put_address_in_context('url') }}:{{ ironic_api_port }}"
 ironic_public_endpoint: "{{ public_protocol }}://{{ ironic_external_fqdn | put_address_in_context('url') }}:{{ ironic_api_port }}"
 
+# Valid options are [ '', redis, etcd ]
+ironic_coordination_backend: "{{ 'redis' if enable_redis|bool else 'etcd' if enable_etcd|bool else '' }}"
```

Comment:
Is there any risk in backporting this? Existing systems with redis or etcd deployed will start to use coordination